docs/PacketFence_OpenWrt-Hostapd_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,152 @@
+OpenWrt BarrierBreaker 14.07 with hostapd Quick Integration Guide
+=================================================================
+:encoding: UTF-8
+:lang: en
+:doctype: book
+
+include::includes/global-attributes.asciidoc[]
+
+About this Guide
+----------------
+This guide has been created in order to help sales engineers, product managers, or network specialists demonstrate the PacketFence capabilities on-site with an existing or potential customer.  It can also provide guidelines to setup a proof of concept for a potential PacketFence deployment using OpenWrt BarrierBreaker 14.07 with Hostapd.
+
+Introduction
+------------
+This guide will provide an example for the configuration of an Open SSID (not crypted) and a Secure SSID (802.1x). You will to install wpad and hostapd. This two SSIDs will do radius authentication against PacketFence.
+
+Assumptions
+-----------
+* You have a configured PacketFence environment with working test equipment
+* Where the management IP of PacketFence will be 192.168.1.10 and has s3cr3t as his radius shared secret
+* You have an acces point with OpenWrt BarrierBreaker 14.07 installed
+
+
+Quick installation
+------------------
+Step 1: Packages installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You can install the packages from the web interface of OpenWrt.
+
+Go to *System -> Software*
+
+First update the repos by clicking the button Update lists.
+
+Then you will have to install the packages of Hostapd and wpad.
+
+Go to the tab 'Available packages' and then search for the package hostapd into the 'Filter:' field.
+
+Click Install the hostapd package, the actual version is 2014-06-03.1-1.
+
+Do the same process for the wpad package version 2014-06-03.1-1.
+
+Step 2: Dynamic VLAN Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Create the file : /etc/config/hostapd.vlan
```

Comment:
On the AP? on PF? You say to login in the AP right after so where do I create this?